pulp_docker/app/models.py

```diff
@@ -192,3 +192,14 @@ class DockerRemote(Remote):
     """
 
     TYPE = 'docker'
+
+
+class DockerDistribution(BaseDistribution):
+    """
+    A docker distribution defines how a publication is distributed by Pulp's webserver.
+
+    All docker distributions are made available at /v2/<base_path>/.
```

Comment:
Concerned that documenting the path here could result in a dependency if changed (or different) where it is actually served.

Reply:
The Docker Registry API has to be served at /v2/. I don't think this is going to change for this version of the API.

Reply:
I still think it adds risk of discrepancy with no value.